auth/src/main/java/org/aerogear/mobile/auth/user/UserPrincipalImpl.java

```diff
@@ -260,4 +267,34 @@ public String getRefreshToken() {
         return refreshToken;
     }
 
+
+    /**
+     * Returns the custom attribute.
+     *
+     * @param attribute the attribute name to retrieve
+     * @return the custom attribute.
+     */
+    @Override
+    public String getCustomAttribute(final String attribute) {
+        String attributeValue = "";
```

Comment:
I think we should return null if the attribute doesn't exist?

Reply:
Yes, makes sense. Updated in https://github.com/aerogear/aerogear-android-sdk/pull/149/commits/31234496cf87fb87aa5f2bff718eaf33c36944d3